internal/database/migrations/migrations.go

```diff
@@ -72,16 +77,29 @@ func Seed(db *gorm.DB, locationFinder location.LocationFinder) error {
 	if err != nil {
 		return err
 	}
-
+	var tags []models.Tag
```

Comment:
We can predeclare the length of the array, no need to use `append` (instead we can just do array indexing)